pyteal/compiler/optimizer/optimizer.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+from typing import List, cast
+
+from pyteal.ir.tealcomponent import TealComponent
+from pyteal.ir.tealop import TealOp
+from pyteal.ir.ops import Op
+
+
+@dataclass
+class OptimizeOptions:
+    """An object which specifies the optimizations to be performed.
+    Args:
+        useSlotToStack (optional): cancel contiguous store/load operations
```

Comment:
@algoidurovic Minor:  Propose making docs match casing of the source code.

```suggestion

        use_slot_to_stack (optional): cancel contiguous store/load operations
```